packages/access-api/src/utils/context.js

```diff
@@ -42,12 +41,12 @@ export function getContext(request, env, ctx) {
     env: config.ENV,
   })
 
-  const keypair = Signer.parse(config.PRIVATE_KEY)
+  const signer = configureSigner(config)
   const url = new URL(request.url)
   const db = new D1QB(config.DB)
   return {
     log,
-    signer: keypair,
+    signer,
```

Comment:
```suggestion
    signer: Signer.parse(config.PRIVATE_KEY).withDID(config.ENV === 'production' ? 'did:web:ucan.web3.storage': `did:web:ucan-${config.ENV}.web3.storage`),
```
can't we just do this ? any error will be caught at dev time or CI.

Reply:
I want the advertised DID to come from env variable. A '.web3.storage' domain won't be the best choice 100% of the times, e.g. in localdev or testing various scenarios. env variable gives me flexibility to get something working. once something works/tests end to end, can narrow down the configurability.